compiler/rustc_codegen_ssa/src/mir/mod.rs

```diff
@@ -192,6 +192,9 @@ pub fn codegen_mir<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>>(
             })
             .collect();
 
+    let (traversal_order, reachable_locals) =
```

Comment:
Likewise.

Reply:
This is and must be reverse postorder. The variable comes from `mono_reachable_reverse_postorder`, which is clear but quite a mouthful for a variable. I'd be happy for a suggestion here.